openedx/openedx-platform
# [ENT-486] Add `ENTERPRISE_SUPPORT_URL` to environment config & version bump edx-enterprise to 0.44.0.

We add `ENTERPRISE_SUPPORT_URL` so we can get it from Django settings in https://github.com/edx/edx-enterprise/pull/171 and beyond...

This has an edx-enterprise **version bump** as well from https://github.com/edx/edx-enterprise/pull/171.

**JIRA tickets**: Meant for [ENT-486](https://openedx.atlassian.net/browse/ENT-486).

**Dependencies**: Meant for https://github.com/edx/edx-enterprise/pull/171 -- the configuration PR is up at https://github.com/edx/configuration/pull/4081

**Sandbox URL**: TBD - sandbox is being provisioned.

**Merge deadline**: Some time around the merge of https://github.com/edx/edx-enterprise/pull/171

**Testing instructions**:

1. Set the variable.
1. Go to the course/program enrollment landing page with a fresh user.
1. Confirm the course/program, but decline data sharing consent.
1. When redirected back to the course/program enrollment landing page, notice the message on the top about being unable to enroll you in the course.
1. The message on the top should have the appropriate support link redirect embedded inside it (hover over / click, and see).

**Reviewers**

**Settings**
```yaml
EDXAPP_ENTERPRISE_SUPPORT_URL: "https://github.com/edx/edx-enterprise/pull/171"
EDXAPP_FEATURES:
  ENABLE_COMBINED_LOGIN_REGISTRATION: true
```
